mungegithub/mungers/testowner/owner.go

```diff
@@ -49,10 +50,31 @@ type OwnerList struct {
 // or else the empty string if none is found.
 func (o *OwnerList) TestOwner(testName string) string {
 	name := normalize(testName)
+
+	// exact mapping
 	owner, _ := o.mapping[name]
+
+	// glob matching
+	if owner == "" {
+		for k, v := range o.mapping {
+			match, err := filepath.Match(k, name)
+			if err != nil {
```

Comment:
The error case only happens with malformed patterns, so there's nothing really interesting about it (it should be caught by the initial owner linter). This would then just become:

    if match, err := filepath.Match(k, name); match { owner = v; break }